lib/hooks/useFetchUserHighlights.ts

```diff
@@ -13,8 +13,7 @@ const useFetchUserHighlights = (username: string) => {
   const pageQuery = page ? `page=${page}` : "";
   const limitQuery = limit ? `&limit=${limit}` : "";
   const { data, error } = useSWR<useFetchUserHighlightsResponse, Error>(
-    `users/${username}/highlights?${pageQuery}${limitQuery}`,
-    publicApiFetcher as Fetcher<useFetchUserHighlightsResponse, Error>
+    `users/${username}/highlights?${pageQuery}${limitQuery}`
```

Comment:
This needs to use the `publicApiFetcher` to connect to the correct API

Reply:
Good catch! 👍